docs/x509.rst

```diff
@@ -695,6 +695,20 @@ General Name Classes
 
         :type: :class:`ObjectIdentifier`
 
+.. class:: OtherName
+
+    .. versionadded:: 0.96
+
+    This corresponds to an "otherName."  An otherName has a type identifier and a value represented in binary DER format.
+
+    .. attribute:: type_id
+
+        :type: :term:`text`
+
+    .. attribute:: value
+
+        :type: :term:`bytes`
```

Comment:
No need to wrap this in term (term requires its appearance in the glossary and causes the docs to fail to build properly).